modules/extension/transform/src/main/java/org/geotools/data/transform/Definition.java

```diff
@@ -44,6 +44,8 @@ public class Definition {
 
     CoordinateReferenceSystem crs;
 
+    String description;
```

Comment:
I was about to say, let's keep this one as InternationalString too... but then AttributeTypeBuilder would not support it, it can only handle description as a String (it builds an international string internally, but does not allow to set one).... what a pain... It should not be hard to switch that field to InternationalString though, and have a parallel setDescription/description accepting an InternationalString directly.